pipeline/mysql/base/kustomization.yaml

```diff
@@ -1,7 +1,9 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+nameprefix: mysql-
```

Comment:
I think we can rename it as `pipeline-mysql-` since other kubeflow components (saying kabit) has mysql, too

Reply:
+1 on the rename. At least until https://github.com/kubeflow/kubeflow/issues/2422 is solved